src/commands/base-command.ts

```diff
@@ -378,7 +380,11 @@ export default class BaseCommand extends Command {
         duration,
         status,
       })
-    } catch {}
+    } catch (err) {
+      debug(`${this.name()}:onEnd`)(
+        `Command: ${command}. Telemetry tracking failed: ${err instanceof Error ? err.message : err?.toString()}`,
+      )
+    }
```

Comment:
Not very related but I ran into this time sink while developing locally and it seems worth fixing